.github/workflows/canister-tests.yml

```diff
@@ -203,8 +213,7 @@ jobs:
           # PRs that used to be green may become red (if the new release broke something). While this is not CI best practice, it's
           # a relatively small price to pay to make sure PRs are always tested against the latest release.
           curl -sSL https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_test.wasm -o internet_identity_previous.wasm
-          # the "rand" features are needed for the captcha crate to compile
```

Comment:
Why is this no longer needed?

Reply:
Meant to leave a note about this, my bad. We don't need to build the CAPTCHA crate at all when building the canister tests, I'm really unsure why this was there before. Maybe at some point the canister tests did depend on the canister build (though don't think so) or it was a case of copy pasta